[DOC] Create documentation of open topics (#559)
saros-project/saros: docs/contribute/open-topics.md

---
title: Open Topics
---

Here you can find:

> * information about topics which are started, but not completed

Reviewer: ```suggestion
* information about topics which were started, but not completed
```

> * ideas and implementations which are never integrated into the main branch, but might be interesting.

Reviewer: ```suggestion
* ideas and implementations which were never integrated into the main branch, but might still be of interest.
```

Therefore this page can be used for finding open and interesting topics which can be continued.

> Please, let us know (e.g. via Gitter) before you continue a topic so we can share our thoughts and experiences.

Reviewer: ```suggestion
Please, let us know (e.g. via our [Gitter](https://gitter.im/saros-project/saros) or [mailing list](https://groups.google.com/forum/#!forum/saros-devel)) before you continue a topic so we can share our thoughts and experiences.
```
